dev-docs/bidders/zmaticoo.md

```diff
@@ -39,3 +39,97 @@ To use this bidder you will need a valid pubId. For further information, please
 | `bapp`               | optional | Block list of applications by their platform-specific exchange independent application identifiers                         | `badv: ["com.blocked"]` | `string array` |
 | `source`             | optional | A Source object that provides data about the inventory source and which entity makes the final decision (See OpenRTB spec) | `source: {}`            | `object`       |
 | `ext`                | optional | An extension object that allows for custom fields and objects to be sent  (See OpenRTB spec)                               | `ext: {}`               | `object`       |
+
+### AdUnit Format Example
+#### AdUnit Format for Banner
+
+```javascript
+ var adUnits = [{
+    code: 'banner-01',
+    mediaTypes: {
+        banner: {
+            sizes: [[320, 50]]
+        }
+    },
+    bids: [{
+        bidder: "zmaticoo",
+        bidId: '100001222',
```

Comment:
please remove bidId - that's not normally something the publisher would hardcode

Reply:
ok